openshift/templates/nodejs.json

```diff
@@ -189,6 +189,26 @@
       "description": "A secret string used to configure the GitHub webhook",
       "generate": "expression",
       "from": "[a-zA-Z0-9]{40}"
+    },
+    {
+      "name": "DATABASE_SERVICE_NAME",
+      "description": "Database service name"
+    },
+    {
+      "name": "MONGODB_USER",
+      "description": "Username for MongoDB user that will be used for accessing the database"
+    },
+    {
+      "name": "MONGODB_PASSWORD",
+      "description": "Password for the MongoDB user"
+    },
+    {
+      "name": "MONGODB_DATABASE",
+      "description": "Database name"
+    },
+    {
+      "name": "MONGODB_ADMIN_PASSWORD",
+      "description": "Password for the database admin user"
```

Comment:
@bparees the reason to have them in the non-db template is that you can easily connect your application to an existing db by filling in the blanks in the Web Console:
![image](https://cloud.githubusercontent.com/assets/88819/8228223/73006424-15ad-11e5-8a25-834ac40972d8.png)

IMO the difference between db and non-db templates should be limited to:
* db template defines a database Service and DeploymentConfig
* db template sets default values for DATABASE_* parameters